_includes/master/charts/calico/templates/calico-node.yaml

```diff
@@ -271,6 +271,9 @@ spec:
               value: "info"
             - name: FELIX_HEALTHENABLED
               value: "true"
+            # Set default felix liveness and readiness health-check port
```

Comment:
This is the default value, right? If so, we probably shouldn't need to specify it here.